packages/backend/src/config/config.staging.ts

```diff
@@ -21,6 +21,7 @@ export function getStagingConfig(): Config {
       starkexApiKey: getEnv('STARKEX_API_KEY'),
       arbitrumAlchemyApiKey: getEnv('ARBITRUM_ALCHEMY_API_KEY'),
       optimismAlchemyApiKey: getEnv('OPTIMISM_ALCHEMY_API_KEY'),
+      ethereumAlchemyApiKey: getEnv('ALCHEMY_API_KEY'),
```

Comment:
That's for now - we should probably have a separate key for ethereum, so that tvl and activity features do not affect each others performance.